testutils/clustermanager/gke.go

```diff
@@ -123,55 +145,46 @@ func (gsc *GKESDKClient) setAutoscaling(project, clusterName, location, nodepool
 	return gsc.Service.Projects.Locations.Clusters.NodePools.SetAutoscaling(parent, rb).Do()
 }
 
-// Setup sets up a GKECluster client.
-// minNodes: default to 1 if not provided
-// maxNodes: default to 3 if not provided
-// nodeType: default to n1-standard-4 if not provided
-// region: default to regional cluster if not provided, and use default backup regions
-// zone: default is none, must be provided together with region
-// project: no default
-// addons: cluster addons to be added to cluster
-func (gs *GKEClient) Setup(minNodes *int64, maxNodes *int64, nodeType *string, region *string, zone *string, project *string, addons []string) ClusterOperations {
-	gc := &GKECluster{
-		Request: &GKERequest{
-			MinNodes:      DefaultGKEMinNodes,
-			MaxNodes:      DefaultGKEMaxNodes,
-			NodeType:      DefaultGKENodeType,
-			Region:        DefaultGKERegion,
-			Zone:          DefaultGKEZone,
-			BackupRegions: DefaultGKEBackupRegions,
-			Addons:        addons,
-		},
-	}
+// Setup sets up a GKECluster client, takes GEKRequst as parameter and applies
```

Comment:
```suggestion
// Setup sets up a GKECluster client, takes GKERequst as parameter and applies
```

Reply:
your suggestion is also missing the letter :)
But I'll update it